From 326adeac6d67ddf2c3d7c059c8a934cfe04d1ed8 Mon Sep 17 00:00:00 2001
From: Christophe Fergeau <cfergeau@redhat.com>
Date: Thu, 29 Mar 2012 14:42:44 +0200
Subject: [PATCH 3/3] Add controller messages for USB redirection

This adds 3 new controller messages:

CONTROLLER_ENABLE_USB (ControllerValue, boolean)
CONTROLLER_ENABLE_USB_AUTOSHARE (ControllerValue, boolean)
CONTROLLER_USB_FILTER (ControllerData, string)

This fixes rhbz #807295
---
 spice/controller_prot.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/spice/controller_prot.h b/spice/controller_prot.h
index f7b1f26..160025a 100644
--- a/spice/controller_prot.h
+++ b/spice/controller_prot.h
@@ -78,6 +78,10 @@ enum {
     CONTROLLER_COLOR_DEPTH,
     CONTROLLER_DISABLE_EFFECTS,
 
+    CONTROLLER_ENABLE_USB,
+    CONTROLLER_ENABLE_USB_AUTOSHARE,
+    CONTROLLER_USB_FILTER,
+
     //spice client -> extrenal app
     CONTROLLER_MENU_ITEM_CLICK = 1001,
 };
-- 
1.7.9.3

